datafusion/src/physical_plan/file_format/parquet.rs

```diff
@@ -59,14 +60,12 @@ use tokio::{
 
 use async_trait::async_trait;
 
-use crate::datasource::{FilePartition, PartitionedFile};
-
 /// Execution plan for scanning one or more Parquet partitions
 #[derive(Debug, Clone)]
 pub struct ParquetExec {
     object_store: Arc<dyn ObjectStore>,
-    /// Parquet partitions to read
-    partitions: Vec<ParquetPartition>,
+    /// List of parquet files, grouped by output partition
```

Comment:
"output partition" is vague here. 
`file_group`, i.e. `Vec<PartitionedFile>`, is the unit of parallelism and will be processed by one single executor/thread.

Reply:
are you suggesting us coming up with a name that's semantically closer to concurrency?

Reply:
I was referring to the `ExecutionPlan.output_partitioning()`. Let me change this for something slightly more explicit 🙂 (I'll try to update this later today)

Reply:
Yes. I think we can rephrase this line to avoid the ambiguity

Reply:
Done!

Reply:
Lgtm